packages/react-google-maps-api/src/utils/injectscript.ts

```diff
@@ -22,17 +26,23 @@ export const injectScript = ({ url, id }: InjectScriptArg): Promise<any> => {
         existingScript.remove()
       }
     }
+    const windowWithGoogleMap: WindowWithGoogleMap = window
+
+    if (document.getElementById(id)) {
```

Comment:
I'm afraid this will reintroduce a bad behavior that was fixed here: https://github.com/JustFly1984/react-google-maps-api/pull/189/files

Reply:
Well, if we enforce a single static ID, there is no problem.

Reply:
There could, because if the url change and we won't create the new script.

But looking at the whole picture: https://github.com/JustFly1984/react-google-maps-api/blob/20194bba212bc641d981d2be9fb679f46f05bf7a/packages/react-google-maps-api/src/utils/injectscript.ts

This line is actually useless, because when going to this line, the script will weither not exist or has been removed, so `document.getElementById(id)` will never return a script